--- a/configure.in	2006-06-10 11:22:48.000000000 +0200
+++ a/configure.in.oden	2008-12-21 16:40:43.000000000 +0100
@@ -395,11 +395,6 @@ else
 fi
 AC_SUBST(XPLIB)
 
-dnl
-dnl Build libDtPrint (in lib/Dt) only if we have libXp and we're building a 2.1 version.
-dnl
-AM_CONDITIONAL(BuildLibDtPrint, test "$LT_HAVELIBXP" = "1" -a "$Build21" = "yes")
-
 AC_PATH_MOTIF
 if test -n "$motif_includes"; then
   if test "x$motif_includes" != "xNONE"; then
